includes/Field/BlockSupports/Anchor.php

```diff
@@ -31,18 +31,12 @@ public static function register( WP_Block_Type $block_spec ) {
 							'type'        => 'string',
 							'description' => __( 'The anchor field for the block.', 'wp-graphql-content-blocks' ),
 							'resolve'     => function ( $block ) {
-								$attribute_config = array(
-									'type'      => 'string',
-									'source'    => 'attribute',
-									'attribute' => 'id',
-									'selector'  => '*',
-								);
 								$rendered_block   = wp_unslash( render_block( $block ) );
 								$value            = null;
 								if ( empty( $rendered_block ) ) {
 									return $value;
 								}
-								$value = DOMHelpers::parseAttribute( $rendered_block, $attribute_config['selector'], $attribute_config['attribute'], null );
+								$value = DOMHelpers::parseFirstNodeAttribute( $rendered_block, 'id' );
```

Comment:
Special case for Anchor. We look only on the first node element for the attribute.